datafusion/sqllogictest/test_files/copy.slt

```diff
@@ -53,32 +53,32 @@ select * from validate_partitioned_parquet_bar order by col1;
 2
 
 # Copy to directory as partitioned files
```

Comment:
I think it would be better to add a new test (not modify an existing test) to show that the existing behavior is not changed